### PR TITLE
Update tic templates [ci-skip]

### PR DESCRIPTION
**Pull request by pat-s**
{tic} templates can be updated :rocket: :robot:

### CODE REVIEW

**Comment by pat-s** [comment]
I'm still somewhat surprised but this automation seems to work quite well 😄

File: .github/workflows/tic.yml [hunk 1]
```diff
@@ -23,55 +23,49 @@ jobs:
         config:
           # Custom configs
           # [Custom matrix env var]
-          - { os: macOS-latest, r: "devel", TIC_ONLY_TESTS: true, TIC_DEV_VERSIONS: true, id: "TIC_ONLY_TESTS & TIC_DEV_VERSIONS", cache: "dev" }
+          - { os: ubuntu-latest, r: 3.6, id: "default" }
```
Comment (krlmlr) [inline .github/workflows/tic.yml:26]: Why is the order reversed here?

Comment (pat-s) [reply .github/workflows/tic.yml:26]: this is due to the iterative approach internally which writes the last item in the loop (here `"custom matrix env var"`) right after the respective anchor (`config:` here).

Comment (krlmlr) [reply .github/workflows/tic.yml:26]: Would sorting help get rid of the problem and also provide some well-defined order in the config here?

Comment (pat-s) [reply .github/workflows/tic.yml:32]: A custom sort enforced by tic would probably not be desired by users?
Right now, the order is reversed but the the internal sequence between the config is kept (at least).

But yeah, thinking about it there would be another reverse going on during the next update via the logic I described above.
Definitely room for improvement.

**Pull request closed without merging**